announcement.md

```diff
@@ -4,7 +4,7 @@ date: 2017-10-17
 summary: TiDB is compatible with MySQL, strong consistent and highly available.
```

Comment:
The new filename is too general. I would use something like `announcing-tidb-1.0-release.md`. See https://pingcap.feishu.cn/docs/doccnIShlpKetyVrDl5tOrSduef#0ArnXY for detailed URL tips.

Reply:
done